paddlenlp/datasets/cote.py

```diff
@@ -64,21 +94,19 @@ def _get_data(self, mode, **kwargs):
 
     def _read(self, filename, split):
         """Reads data"""
-        _, _, field_indices, num_discard_samples = self.BUILDER_CONFIGS[
-            self.name]['splits'][split]
         with open(filename, 'r', encoding='utf-8') as f:
             for idx, line in enumerate(f):
-                if idx < num_discard_samples:
+                if idx == 0:
+                    # ignore first line about title
                     continue
                 line_stripped = line.strip().split('\t')
                 if not line_stripped:
                     continue
-                example = [line_stripped[indice] for indice in field_indices]
                 if split == "test":
-                    yield {"tokens": list(example[0])}
+                    yield {"tokens": list(line_stripped[1])}
                 else:
                     try:
```

Comment:
It might be better to use line.split('\t', 1) to handle lines who have more than one \t. And use len(line_stripped) and split to handle dirty data which is seperated in two lines.